src/main/java/org/sqlflow/client/model/RequestHeader.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2019 The SQLFlow Authors. All rights reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.sqlflow.client.model;
+
+import lombok.Data;
+
+@Data
+public class RequestHeader {
```

Comment:
Looks like the generated Java proto file is at `proto.Sqlflow.Request` and `proto.Sqlflow.Session`. Can we omit to redefine RequestHeader and use the proto file directly?